### ISSUE TITLE
:hammer_and_pick: One, global `env`

### ISSUE THREAD

**Issue by jadudm** [jadudm/eight#2]
# Background

I initially thought `env` work should be pushed to the periphery---as  in, per-component. 

I now think it should be one, central, global `env`. It should be loaded at app-start, and it should have everything needed.

This will work for individual services, and it will work if multiple services get bundled into a single app.

Passing `env` around makes all the code more complex; I'll risk the fragility of a global env for the simplicity of the resulting code.

### Tasks
- [ ] Consider `viper` for env/configuration management.
- [ ] Create a `cfenv.json` (or, use `viper`) to create a configuration locally that mimics CF's environment.
- [ ] Write an ADR for the decision.

## Initial thinking

The app wants to run:

1. Locally (for dev/test)
2. Locally (in container, for accuracy)
3. In CloudFoundry

Do I need all three? It is easier to do environment handling in 2 & 3, because it is consistent/repeatable.

In short:

1. Settle on where the app can run. (Perhaps it is always containerized)
2. Rethink env handling

The second point is that I want the `env` to operate "the same way" when local and remote. So, everything locally needs to come from a fake `VCAP_SERVICES` and friends.

**Comment by jadudm** [comment]
:shipit: There should be one, top-level env.

**Comment by jadudm** [comment]
Blocked by #6 .

**Comment by jadudm** [comment]
closed via https://github.com/jadudm/eight/pull/9
